README.md

```diff
@@ -44,27 +60,27 @@ Next, check **[First Time Setup](#first-time-setup)** to continue setting up you
 
 #### Connect your store with your Omise account
 
-![Omise-WooCommerce plugin - setting page](https://user-images.githubusercontent.com/2154669/62671292-7b991480-b9c0-11e9-8627-0bc85c078365.png)
+![Omise-WooCommerce plugin - setting page](https://user-images.githubusercontent.com/2154669/68251435-b3face00-0055-11ea-9932-d2129bc3f1e7.png)
 
 Once the installation is done, the next thing that you are going to do is to connect your store with your Omise account and enable payment methods so your customers can make a purchase with an online payment.
 
 1. Log in to WordPress admin page.
 
 2. From the sidebar, go to `Omise > Settings`.
 
-3. At the Omise Settings page, you are going to set your `Public key` and `Secret key` (these keys can be found at Omise Dashboard).
+3. At the Omise Settings page, you are going to set your `Public key` and `Secret key` (these keys can be found at [Omise Dashboard](https://dashboard.omise.co/test/keys)).
 
 4. Make sure that the option `Test mode` is checked and set your Omise keys at `Public key for test` and `Secret key for test` fields if you would like to test Omise service integration.
 
 5. Click **'Save Settings'**.
 
 #### Enable payment methods
 
-> Note that Alipay, Bill Payment: Tesco, Installment, and Internet Banking payment methods are only available for merchants with a Thai-registered Omise account.
+> Note that Alipay, Bill Payment: Tesco, Installment, Internet Banking, and TrueMoney Wallet payment methods are only available for merchants with a Thai-registered Omise account.
 
-There are 5 payment methods that you can enable on your store, which are **Alipay**, **Bill Payment: Tesco**, **Credit / Debit Card**, **Installment**, and **Internet Banking**.
+There are 6 payment methods that you can enable on your store, which are **Alipay**, **Bill Payment: Tesco**, **Credit / Debit Card**, **Installment**, **Internet Banking**, and **TrueMoney Wallet**.
 
-1. At Omise Settings page, Payment Methods section, there will be 5 available payment methods show on the list.
+1. At Omise Settings page, Payment Methods section, there will be 6 available payment methods show on the list.
```

Comment:
Replace 6 with 'a number of'
'show on the list' should be 'shown on the list'
'At Omise Settings page' should probably be  'At the Omise Settings page'